Mark Forecast as EOL

yakushabb opened the pull request:
The project archived by the developer.

See: https://github.com/SalaniLeo/Forecast

> This repository was archived by the owner on Sep 24, 2025.
> It is now read-only.

yakushabb committed:
Mark Forecast as EOL

The project archived by the developer.

See: https://github.com/SalaniLeo/Forecast

> This repository was archived by the owner on Sep 24, 2025.
> It is now read-only.

ghost commented:
@barthalion I've marked already archived Forecast app as EOL.

Could you review changes.

Also @SalaniLeo, FYI

yakushabb committed:
Fix a build error

> 'manifest-unknown-properties' error found in linter manifest check.
> Details: Invalid property 'dependencies' in context 'module'

SalaniLeo commented:
oh yeah thank you I forgot to mark as eol when I archived the repo. sorry!
everything looks ok, I'll merge
